base/commandLine.cpp

```diff
@@ -752,6 +756,24 @@ static void listAudioDevices() {
 	}
 }
 
+/** Display all games in current directory */
+static void autoDetect(){
+	//Current directory
```

Comment:
Could you explain about comments, code to which the comment must allocated by blank lines?